app/src/main/java/com/example/bigowlapp/model/Notification.java

```diff
@@ -9,8 +9,13 @@
 
     @DocumentId
     private String uId;
+    private String senderUId;
+    private String receiverUId;
+    private String groupUId;
     private String type;
-    private Timestamp time;
+    private Timestamp timeRead;
+    private Timestamp timeSend;
+    private Schedule.UserResponse senderResponse;
```

Comment:
You should extract UserResponse to its own class instead of using it as a inner class of Schedule if its needed in this class